package/MDAnalysis/transformations/__init__.py

```diff
@@ -53,7 +53,30 @@ def wrapped(ts):
 
 See `MDAnalysis.transformations.translate` for a simple example.
 
+To meet the need of serialization of universe, transformations are converted
+into classes. They retain the similar API and functionality by implementing the
+aforementioned `wrapped(ts)` as a special method `__call__`. For example:
 
+.. code-blocks:: python
+
+    class transfomrations(object):
+        def __init__(self, *args, **kwargs):
+            #  do some things
+            #  save needed args as attributes.
+            self.needed_var = args[0]
+
+        def __call__(self, ts):
+            #  apply changes to the Timestep object
+            return ts
+
+
+Note it does not mean that the old closure/wrapped function implementation will fail.
+They can still be used if one does not need serialization, which is a prerequisite for parallel
+analysis.
```

Comment:
If a new reader reads this, will they know what "old" means? This should be written from the perspective of the present time. You might have to explain that you can also write functions (closures) as in MDA 1.x but that these cannot be serialized and thus will not work with all forms of parallel analysis.